src/main/main.ts

```diff
@@ -90,10 +94,30 @@ const createWindow = async () => {
     } else {
       mainWindow.show();
     }
+
+    pyServer = startServer();
+
+    if (pyServer !== null && pyServer.stderr !== null) {
+      pyServer.stderr.once('data', (data) => {
+        console.log(data.toString()); // prints address that the server is running on. TODO: remove before it is electron app is built
+        get('http://localhost:5000', (res) => {
+          if (res.statusCode === 200) {
+            console.log('Python server is running');
+          } else {
+            throw new Error(
+              `Python server has an error, response to a get '/' expected code 200 got ${res.statusCode}`
+            );
+          }
+        });
+      });
+    }
   });
 
   mainWindow.on('closed', () => {
     mainWindow = null;
+    if (pyServer !== null && typeof pyServer.pid === 'number') {
```

Comment:
Why do we need to check the process ID's type?

Reply:
Oops, removed it.